stock_landed_costs_average/model/stock_landed_costs.py

```diff
@@ -587,7 +588,13 @@ def compute_landed_cost(self, cr, uid, ids, context=None):
                         else:
                             value = (line.price_unit / total_line)
 
-                        if digits:
+                        # When the valuation is the last record, assign
+                        # line.price_unit less the accumulated value in
+                        # value_split
+                        if digits and \
+                                valuation == cost_adjust_lines[-1]:
```

Comment:
@JulioSerna This way you make sure that the remaining amount and the all previous amount that have already been allocated will sum as line.price_unit.

for example:

if you have line.price_unit = 100.

and there are three lines when divided: 100 / 3 = 33.33

the last one will have to be 33.34, 

using this approach will ensure this will happen.

Regards